modules/javafx.graphics/src/main/java/javafx/css/Selector.java

```diff
@@ -40,12 +43,14 @@
  *
  * @since 9
  */
-abstract public class Selector {
+public abstract sealed class Selector permits SimpleSelector, CompoundSelector {
 
     /**
-     * Package scoped constructor.
+     * Explicit constructor for subtype use.
+     *
+     * @since 23
```

Comment:
The pattern we use elsewhere is:

"Constructor for subclasses to call."

Also, that should be `@since 24`.